### DISCUSSION TITLE
Different location to fetch chunks

### DISCUSSION THREAD

**Discussion by AsuraKev** [web-infra-dev/modern.js#4656] [Q&A] [answered]
Hi there, I am trying to convert a vite project to modern js. In vite we can configure the app to fetch js chunks from a url e.g https://domain/js/chunk/212312.js  rather than from the root /

any idea how to do that in Modern js?

thanks

**Comment by chenjiahan** [comment, marked as answer]
You can use the `assetPrefix` config.

- dev.assetPrefix: https://modernjs.dev/en/configure/app/dev/asset-prefix.html
- output.assetPrefix: https://modernjs.dev/en/configure/app/output/asset-prefix.html
